src/aat/java/uk/gov/hmcts/reform/dg/docassembly/functional/OpenIdConnectSceanarios.java

```diff
@@ -19,100 +16,87 @@ public class OpenIdConnectSceanarios extends BaseTest {
     @Rule
     public ExpectedException exceptionThrown = ExpectedException.none();
 
-    @Rule
-    public RetryRule retryRule = new RetryRule(3);
```

Comment:
I remember adding this Retry for 3 times. As the Nightly Build tends to be flaky. So not sure if we want to remove this.